source/uwp/ObjectModelProjection/ObjectModelProjection.vcxproj

```diff
@@ -233,6 +233,10 @@
       <WarningLevel>Level4</WarningLevel>
       <TreatWarningAsError>true</TreatWarningAsError>
       <DisableSpecificWarnings>4702</DisableSpecificWarnings>
+      <ControlFlowGuard Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">Guard</ControlFlowGuard>
```

Comment:
nit: will it be simple if
`<ControlFlowGuard>Guard</ControlFlowGuard>`
or 
`<ControlFlowGuard Condition="'$(Configuration)'=='Release'">Guard</ControlFlowGuard>`

Reply:
Yup, if we're enabling for all the Release flaword, simply set it to `<ControlFlowGuard Condition="'$(Configuration)'=='Release'">Guard</ControlFlowGuard>`

Reply:
Just so folks know for future reference -- the pattern of change as submitted is what happens when you edit the project file through the VS UI. It's not really necessary to change it, as it would only be a problem when adding new architectures to our build, which would require a manual review of all of this anyways. If we add an arch through VS, it's likely that it would make similar changes there as well.

Reply:
@paulcam206 thanks, that's nice to know :)
I've never edited project through VS UI (always manually) and I had no idea that it's what it does by default:)